framework/src/utils/PetscSupport.C

```diff
@@ -885,7 +889,10 @@ colorAdjacencyMatrix(PetscScalar * adjacency_matrix,
   }
 
   MatDestroy(&A);
+#if PETSC_VERSION_LESS_THAN(3, 5, 0)
+#else
```

Comment:
Replace ```
 #if PETSC_VERSION_LESS_THAN(3, 5, 0)
             #else
```
with 

`#if !PETSC_VERSION_LESS_THAN(3, 5, 0)`

Reply:
Addressed in commit a3b8bab.